src/Directory.Build.props

```diff
@@ -5,4 +5,9 @@
     <Nullable>enable</Nullable>
   </PropertyGroup>
 
+  <ItemGroup>
+    <PackageReference Include="TunnelVisionLabs.ReferenceAssemblyAnnotator" Version="1.0.0-alpha.154" PrivateAssets="all" />
+    <PackageDownload Include="Microsoft.NETCore.App.Ref" Version="[3.1.0]" />
+  </ItemGroup>
```

Comment:
This brings two significant benefits:

- It reduces the number of times you have to add nullability suppressions (`!` operator) because it clues the compiler in to e.g. the fact that `string.IsNullOrEmpty` is a null check.
- It catches places where the BCL is documented to sometimes return null which would cause Shouldly to throw NullReferenceException.